### ISSUE TITLE
Clear out the cruft in api.js

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/monarch-legacy#975]
There are vast swathes of api.js that are unused. Anything that calls anything that uses federation_services_url can go

caveat: landing pages still have a dependency. @kshefchek had the idea of just checking in the json cache for these as a temporary solution.

**Comment by jmcmurry** [comment]
@kshefchek any update? What exactly would be needed to close this?

**Comment by kshefchek** [comment]
I've taken the approach of commenting out portions and then deleting if all seems fine.  In the literature pages PR I commented a good chunk of the api.js code, and at this point it's probably safe to delete it.  That said there's still a bit more than can be cleaned up on top of this.

**Comment by cmungall** [comment]
Yes, delete commented out code. This is good practice anyway, we have memory in git.

I'm not sure how easy it is to see a call tree in javascript, but `fetchDataFromResource` must die, and along with it any code that calls it. 

Similarly, any method name with ontoquest in it.

Note the landing pages no longer have a dependency, so this should be much easier

We can also kill anything for ringo

Although this will not affect functionality, it's important for understanding our own codebase and moving forward with future refactors.

I don't think it should take long

**Comment by cmungall** [comment]
I'll do this

**Comment by cmungall** [comment]
@kshefchek is generateHistogram used? It has a bunch of dependencies to federation

**Comment by cmungall** [comment]
I believe these are no longer used:

```
web.wrapRouteGet(app, '/gene/:id/:section.:fmt?', '/gene/{id}/{section}.{fmt?}', ['id', 'section', 'fmt'], fetchFeatureSection);
web.wrapRouteGet(app, '/variant/:id/:section.:fmt?', '/variant/{id}/{section}.{fmt?}', ['id', 'section', 'fmt'], fetchFeatureSection);
web.wrapRouteGet(app, '/genotype/:id/:section.:fmt?', '/genotype/{id}/{section}.{fmt?}', ['id', 'section', 'fmt'], fetchFeatureSection);
web.wrapRouteGet(app, '/model/:id/:section.:fmt?', '/model/{id}/{section}.{fmt?}', ['id', 'section', 'fmt'], fetchFeatureSection);
```

I will get these on the next pass

**Comment by kshefchek** [comment]
@cmungall no it has been replaced, feel free to remove.
